doc/source/reference/executables/groonga-httpd.rst

```diff
@@ -543,6 +543,41 @@ configuration, run the following command to remove cache files::
 If you use Groonga's query cache feature, you don't need to expire
 cache by hand. It is done automatically.
 
+Nginx log related directives
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``access_log``
+""""""""""""""
+
+We can output Nginx's log by the ``access_log`` directive.
+The following example is default Nginx's log settings::
+
+  http {
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
```

Comment:
Why do you use `main` as format name?
The default format is called `combined` in nginx.